labs/a3.md

```diff
@@ -78,14 +78,21 @@ sudo apt install --no-install-recommends qemu-kvm qemu-utils libvirt-bin virtins
 
 ### acquire installation media
 
-Next, we need to get the installation media. Download this, as well as
-a signature that we will use to verify the authenticity of the media,
-using the following commands:
+Next, we need to get the installation media. Go to
+https://mirrors.ocf.berkeley.edu/archlinux/iso/latest/ to find the latest Arch
+release. Using the `wget` command within your VM, download the `.iso` file found
+within this directory as well as the `.iso.sig` signature file that we will use
+to verify the authenticity of the `.iso` file. For example, the current release
+at the time of writing is `2020.06.01`, so you would run the following commands:
 
 ```sh
-wget 'https://mirrors.ocf.berkeley.edu/archlinux/iso/2019.09.01/archlinux-2019.09.01-x86_64.iso'
-wget 'https://mirrors.ocf.berkeley.edu/archlinux/iso/2019.09.01/archlinux-2019.09.01-x86_64.iso.sig'
+wget 'https://mirrors.ocf.berkeley.edu/archlinux/iso/latest/archlinux-2020.06.01-x86_64.iso'
+wget 'https://mirrors.ocf.berkeley.edu/archlinux/iso/latest/archlinux-2020.06.01-x86_64.iso.sig'
 ```
+Note that the Arch distribution is updated relatively frequently, so the links
```

Comment:
Would it be relevant to discuss briefly about how arch is rolling release and what that entails?

Reply:
Added rolling release explanation and compared it to Ubuntu release cycle.